### DISCUSSION TITLE
What does the Error "Move abort 0x6000d" means? Why custom asserts are not working ?

### DISCUSSION THREAD

**Discussion by Nishu0** [aptos-labs/aptos-developer-discussions#115] [Questions] [answered]
I have placed a few custom error checks with assert ```assert!(
            exists<SeniorCapabilities>(mod_account),
            error::not_found(ENO_CAPABILITIES),
        );``` like this in a few places in my code. But almost whenever I want to know where the error is coming from it gives a generalized  error like this "Move abort 0x6000d in my_module_address::lending_vaultundefined"

**Comment by kshitijc1506** [comment]
You want to know which account address is responsible for the error (because the SeniorCapabilities resource doesn't exist in that account). Is this an accurate description of what you're trying to achieve?

Could you provide a code snippet where you encountered this error, and/or share the transaction hash if the error was observed in a failed transaction?

**Comment by gregnazario** [comment, marked as answer]
## Error messages

Error messages will only show up if you put a doc comment above the constant:

```move

/// My error message will show up
const E_MY_ERROR: u64 = 1;
```

Which SDK, CLI or tool are you using that is giving you this error, and what version is it?
